tests/test_form.py

```diff
@@ -546,5 +546,43 @@ def test_option_without_value(fail, selected, expected_post):
         assert res.status_code == 200 and res.text == 'Success!'
 
 
+one_field_form = '''<form method="post">
+  <input name="life" type="hidden" value="good"/>
+</form>'''
+
+
+def test_get_value_of_input_field():
+    """Test that we can get input field of current form by name"""
+
+    browser, url = setup_mock_browser(text=one_field_form)
+    browser.open(url)
+    browser.select_form()
+
+    assert browser['life'] == 'good'
```

Comment:
While you're there, checking that assignment assigns to the same place as reading would be nice:

```suggestion
    assert browser['life'] == 'good'
    browser['life'] = 'veryGood'
    assert browser['life'] == 'veryGood'
```